packages/desktop/views/dashboard/vesting/Vesting.svelte

```diff
@@ -31,8 +35,14 @@
         MeatballMenuButton,
         TooltipIcon,
     } from '@ui'
+    import { onMount } from 'svelte'
 
     const DEFAULT_EMPTY_VALUE_STRING = '----'
+    let modal: Modal
+    let timeUntilNextPayout = DEFAULT_EMPTY_VALUE_STRING
+    let minRequiredStorageDeposit: number
+    let hasOutputsToConsolidate = false
```

Comment:
I would create a variable called "canCollect" or something like that which has the logic in a reactive way, so we dont have to add the logic in the html section, we now have `{$selectedAccountVestingUnclaimedFunds === 0 ||
                                        hasTransactionInProgress ||
                                        $selectedAccount?.balances?.baseCoin?.available < minRequiredStorageDeposit ||
                                        hasOutputsToConsolidate === false` which is getting a bit out of hand imo 🙏🏼